### PR TITLE
Moved Router to a Seperate File.

### PR DESCRIPTION
**Pull request by sanchit94**

### CODE REVIEW

**Comment by atherdon** [comment]
we have issues in two tests :)
you see - this is why i love the tests

File: src/component/FormFields/Attribute.js [hunk 1]
```diff
@@ -8,12 +8,15 @@ import data from '@groceristar/groceristar-fetch/search';
 let attribute;
 
 function toOpt(arr) {
-    let Opt= arr.reduce((intermediate, item, index) => {
+    let Opt
+    if(typeof arr === 'object') {
+        Opt = arr.reduce((intermediate, item, index) => {
```
Comment (atherdon) [inline src/component/FormFields/Attribute.js]: what do you think about installing lodash and using it for this type of functions? 
i think it'll make it less complicated

Comment (sanchit94) [reply src/component/FormFields/Attribute.js]: Well, I haven't used lodash so I'll have to look it up at the documentation.

Comment (atherdon) [reply src/component/FormFields/Attribute.js]: sure, no prob. you'll like it.
i'll merge this PR, but later when you'll have time - check that lib. will create a separated issue for it

Comment (sanchit94) [reply src/component/FormFields/Attribute.js]: Sure! :)

**Pull request merged**